notebooks/resources/utils.py

```diff
@@ -139,16 +139,16 @@ def create_s3_buckets():
 
 def init_rsclient(owner_id=None, cadip_station=ECadipStation.CADIP):
     """Init RsClient instances"""
-    global apikey, auxip_client, cadip_client, stac_client
+    global apikey, auxip_client, cadip_client, stac_client, staging_client
 
     # In local mode, the service URLs are hardcoded in the docker-compose file
     if local_mode:
         rs_server_href = None  # not used
-
+        rs_server_href_staging = None
     # In cluster mode, they are set in an environment variables
     else:
         rs_server_href = os.environ["RSPY_WEBSITE"]
-
+        rs_server_href_staging = "https://rsserverstaging.dev-rspy.esa-copernicus.eu"
```

Comment:
@agrosu1978 do you know why we need this utl and not https://dev-rspy.esa-copernicus.eu ?

Reply:
@mcolin16 after Nicolas' fix yesterday, we don't need this anymore ? Idem for line 147 above.